lib/commands/plugin/fetch-plugin.ts

```diff
@@ -22,13 +22,14 @@ export class FetchPluginCommand implements ICommand {
 				this.$logger.out(result);
 			} else {
 				var plugins = this.$cordovaPluginsService.getPlugins(args);
-				var pluginsCount = Object.keys(plugins).length;
+				var pluginKeys = Object.keys(plugins);
+				var pluginsCount = pluginKeys.length;
 				if (pluginsCount === 0) {
 					this.$logger.out("There are 0 matching plugins.");
-				} else if (pluginsCount > 1) {
+				} else if (pluginsCount > 1 && pluginKeys[0] != args[0]) {
```

Comment:
!==
also are you sure that the plugins are sorted, for example if you have:
plugin, plugin1, plugin2
Are you sure they'll always be in this order? In case they are plugin1, plugin, plugin2 and you search for plugin, this check will fail.